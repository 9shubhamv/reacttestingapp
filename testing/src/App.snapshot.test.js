import React from "react";
import renderer from 'react-test-renderer';
import App from "./App";

it('SnapShot testing of App Component',()=>{
    const tree=renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
})