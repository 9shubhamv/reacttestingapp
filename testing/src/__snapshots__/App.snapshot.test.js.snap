// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapShot testing of App Component 1`] = `null`;
